src/workloads.jl

```diff
@@ -130,10 +130,8 @@ macro setup_workload(ex::Expr)
         :((ccall(:jl_generating_output, Cint, ()) == 1 && $PrecompileTools.workload_enabled(@__MODULE__)))
     end
     return esc(quote
```

Comment:
```suggestion
    # Ideally we'd like a `let` around this to prevent namespace pollution, but that seem to
    # trigger inference & codegen in undesirable ways (see #16).
    return esc(quote
```